tasks/slack-notify.js

```diff
@@ -1,120 +1,154 @@
-var gulp = require('gulp');
-var os = require('os');
-var gutil = require('gulp-util');
-var request = require('superagent');
-var argv = require('yargs').string('rev').argv;
-
-var getRevision = require('./utils').getRevision;
-var getConfigFor = require('./utils').getConfigFor;
-var getFullName = require('./utils').getFullName;
+const gulp = require('gulp');
```

Comment:
Not yet. I included just syntax changes in this PR. I will be looking into bugs in a separate PR.